docs/book/localization.md

```diff
@@ -464,52 +462,213 @@ backslash:
+[localization] +0ms - Hello.|Hi.
+```
+
+And `localization.json` would now have:
+
+```json
+"speech": {
+  "Howdy.": {}, // new line
+  "What's your name?": {
+    "es": "¿Cómo se llama?"
+  },
+  "Hello.|Hi.": { // orphaned line
+    "es": "Hola."
+  }
+}
+```
+
+So, if the translations from the original string still apply, they should be
+copied to the new string mapping manually.
+
+Intents and utterances changed in the same way follow the same behavior.
+
+### `litexa localize` arguments
 
 `litexa localize` can receive some arguments to modify its behavior. To see the
 complete list, run `litexa localize --help`.
 
-#### Cloning
+#### Verbose logs
+
+The `--verbose` flags turns on logging to give more detail on the contents that
+have changed from the existing `localization.json`.
+
+#### Auto-removing orphaned strings
+
+Orphaned speech strings and orphaned utterances can be automatically removed
+with the `--remove-orphaned-speech` and `--remove-orphaned-utterances` flags,
+respectively, but note that doing so will also remove their translations from
+`localization.json`.
+
+#### Cloning translations
 
-With cloning, the arguments for cloning your default language to French would look like:
+With cloning, the arguments for cloning your default language to French would
+look like:
 
 ```bash
-litexa localize --clone-from default --clone-to fr
+litexa localize --clone-from fr-FR --clone-to fr-CA
 ```
 
+This can be useful as a starting point for adjusting translations for a
+different region.
+
 :::warning Cloning to an existing language
 If you clone to a language that already exists, you will overwrite its existing
 localization in the localization file.
 :::
 
 ## Combining both localization strategies
+
 You can combine both localization methods in accordance with your use case
 (e.g, project structure-based overrides for assets, and string replacement map
 for dialogue). However, they function completely independently of each other.
 Also, remember: the string replacement map method only extracts information from the
 default language.
 
+## Localization in code
+
+Both localization methods do not cover all cases of localizing strings. 
+this means that [slotbuilder functions](/book/state-management.html#slots),
+`context.say.push` calls, function calls that may use strings, and inline code
+speech injection are not localized.
+
+In your Litexa files, you can pass `context.language` to code to retrieve the
+Litexa language.
+
+As an example, let's assume your Litexa code had a custom color slot type and
+an inline function that injects speech directly into `context.say`:
+
+```coffeescript
+# main.litexa
+launch
+  when "$color"
+    with $color = mySlotBuilders.js:myColorSlotBuilder
+    injectSpeech(context)
+```
+
+The inline code could use `context.language` to do any necessary localization.
+
+```javascript
+// inlineCode.js
+function injectSpeech(context) {
+  switch (context.language) {
+    case 'fr':
```

Comment:
Do we mention using a region-agnostic language code anywhere? If not, we should maybe use something that is more aligned with the other examples in here:

```
case 'fr-CA':
case 'fr-FR':
```

Reply:
Yea - there's a tip in the project structure section for this and an example above it in the same section.